src/Cygni.Snake.Client/DirectionalResult.cs

```diff
@@ -2,10 +2,10 @@
 {
     public enum DirectionalResult
     {
-        Nothing,
-        Food,
-        Death,
-        Danger,
-        TailNibble
+        Food = 1,
```

Comment:
The  DefaultBackupBot assumes these are in ascending order of danger. Perhaps worth adding a comment mentioning that any new results should stay true to that rule.